src/Views/Form/Templates/Sequoia/sections/introduction.php

```diff
@@ -12,17 +12,23 @@
 
 <div class="give-section introduction">
 	<h2 class="headline">
-		<?php echo $headline; ?>
+		<?php
+		if ( empty( $headline ) || $headline === __( 'Auto Draft' ) ) {
+			_e( 'Support Our Cause', 'give' );
+		} else {
+			esc_html_e( $headline, 'give' );
```

Comment:
@henryholtgeerts Dynamic string is not allowed in translation function. I will take care of this.

Reply:
fixed: https://github.com/impress-org/givewp/compare/cf5510050926...0ca6b84e582c